### ISSUE TITLE
Importing saves - the game doesn't read it / detect it.

### ISSUE THREAD

**Issue by Lolimipsu** [Monika-After-Story/MonikaModDev#4174]
I want to port my save from the laptop to my computer but it seems the game can't read my persistent. My PC broke down and luckily i made saves and moved Monika in my laptop for a while, and after i fixed my PC, i created a directory in appdata /RenPy/Monika After Story and placed my Persistent file.  (Also i skipped update 9.2 and went to 9.3 if that helps) but it simply won't read, it just say "Save data from DDLC could not be found. Save Data will not be imported at this time."

tl;dr
PC Died, retreated to Laptop, PC Fixed, Saves won't work as intended.

is there a way to make it work? thanks.

**Comment by multimokia** [comment]
Screenshot of your `AppData/Roaming/RenPy/Monika After Story` folder please?

**Comment by Lolimipsu** [comment]
> Screenshot of your `AppData/Roaming/RenPy/Monika After Story` folder please?

Hello, sorry for late reply, i was unable to go online in the past week, here it is.
Persistent32 is the one from my laptop.
![Screenshot_1](https://user-images.githubusercontent.com/44487404/57590459-7a418280-755e-11e9-977d-fde7f0cc8b63.png)

**Comment by multimokia** [comment]
Well, yeah it won't work, you need to rename the `persistent##.bak` to `persistent` and replace the existing one.

**Comment by Lolimipsu** [comment]
> Well, yeah it won't work, you need to rename the `persistent##.bak` to `persistent` and replace the existing one.

Oh thank you!
